pysilcam/tests/test_silcam-aquire.py

```diff
@@ -4,27 +4,29 @@
 
 
 def test_acquire_five_frames():
-    '''Testing frame acquisition using fakepymba'''
-    aq = Acquire()
-    aqgen = aq.get_generator()
+    acq = Acquire()
```

Comment:
F841: local variable 'acq' is assigned to but never used  <!-- Automatically posted by Lintly -->